server/src/test/java/io/druid/server/audit/SQLAuditManagerTest.java

```diff
@@ -135,6 +134,90 @@ public void testFetchAuditHistory() throws IOException
     Assert.assertEquals(entry, auditEntries.get(1));
   }
 
+  @Test
```

Comment:
Note this was really meant as a general comment and not really meaning "in this exact file"